src/shared/routeHandlers/BaseHandler.js

```diff
@@ -18,12 +13,8 @@ import '../base.scss';
 class BaseHandler extends Component {
   componentDidMount() {
     const { fetchDefinition } = this.props;
-    // TODO: find a way to inject platformSwaggerUrl from definitionViewer into this project
-    const platformSwaggerUrl = 'data/platform-swagger.json';
-    const query = qs.parse(window.location.search.slice(1));
-    this.url = query.url || platformSwaggerUrl;
-
-    fetchDefinition(this.url);
+    this.swaggerURL = this.props.location.query.url;
```

Comment:
`BaseHandler` is a child of `Route`, and thus gets injected this property by default, as documented on [react router docs](https://github.com/ReactTraining/react-router/blob/master/docs/API.md#route).

I've added `location` under `propTypes` for this component to make this requirement explicit.